src/instrumentation/kv.ts

```diff
@@ -1,62 +1,58 @@
 import { Attributes, SpanKind, SpanOptions, trace } from '@opentelemetry/api'
+import { SemanticAttributes } from '@opentelemetry/semantic-conventions'
 import { wrap } from '../wrap.js'
 
 type ExtraAttributeFn = (argArray: any[], result: any) => Attributes
 
+const dbSystem = 'Cloudflare KV'
+
 const KVAttributes: Record<string | symbol, ExtraAttributeFn> = {
 	delete(argArray) {
-		return {
-			'kv.key': argArray[0],
-		}
+		return {}
 	},
 	get(argArray) {
-		const attrs: Attributes = {
-			'kv.key': argArray[0],
-		}
+		const attrs: Attributes = {}
 		const opts = argArray[1]
 		if (typeof opts === 'string') {
-			attrs['kv.type'] = opts
+			attrs['db.cf.type'] = opts
 		} else if (typeof opts === 'object') {
-			attrs['kv.type'] = opts.type
-			attrs['kv.cacheTtl'] = opts.cacheTtl
+			attrs['db.cf.type'] = opts.type
+			attrs['db.cf.cache_ttl'] = opts.cacheTtl
 		}
 		return attrs
 	},
 	getWithMetadata(argArray, result) {
 		const attrs = this.get(argArray, result)
-		attrs['kv.withMetadata'] = true
+		attrs['db.cf.metadata'] = true
 		const { cacheStatus } = result as KVNamespaceGetWithMetadataResult<any, any>
 		if (typeof cacheStatus === 'string') {
-		  attrs['kv.cacheStatus'] = cacheStatus
+		  attrs['db.cf.cache_status'] = cacheStatus
 		}
 		return attrs
 	},
 	list(argArray, result) {
 		const attrs: Attributes = {}
 		const opts: KVNamespaceListOptions = argArray[0] || {}
-		const { cursor, limit, prefix } = opts
-		attrs['kv.list_prefix'] = prefix || undefined
```

Comment:
Any reason you remove the `prefix`?

Reply:
Oh I see, because you added it to the DB_STATEMENT below? Don't you think there is value in having it separate to that?

Reply:
🤔 Probably most uniform would be to use (what is currently in this pr as) `db.cf.key` on ln 85
There is a big advantage for storage to limiting the overall number of distinct attribute keys produced so unifying on `db.cf.key` for the prefix for lists and the id for deletes/gets I think accomplishes this while leaving the info there
Does this work for you?

Reply:
I missed that you added it to the `key` one.. I do not have a strong preference either way. So let's just go with that..